src/lib/TermBuilder.ml

```diff
@@ -225,7 +225,7 @@ let code_sg mbase mfam : _ m =
 let code_path mfam mbdry : _ m =
   let+ fam = mfam
   and+ bdry = mbdry in
-  S.CodePath (fam, bdry)
+  S.CodeExt (1, fam, `Global (S.Lam (`Anon, S.Cof (Cof.Join [S.Cof (Cof.Eq (S.Var 0, S.Dim0)); S.Cof (Cof.Eq (S.Var 0, S.Dim1))]))), bdry)
```

Comment:
@jonsterling could this be refactored with cof_to_con and Cof.boundary?

Reply:
@favonia Note that these are all syntax, not domain. So no.

Reply:
:sob: